### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Blob File_BlobContainerAcquireLease_ScreenReader:NVDA does not announce lease status changes or current locked status for selected blobs.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1906]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #Win32;

**Environment Details:**
Desktop App: Windows Version: Windows10
NVDA Version: 2019.1.1

**Steps to Reproduce:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Right click on blob and select "Acquire Lease" control and hit enter.
7. Open NVDA and check whether it reads anything about the locked status for which do the Acquire lease.
 **Actual:**
NVDA does not read about the locked status for which after click on the acquire lease.
Same issue is observed in JAWS also.
**Expected:**
NVDA should provide the information about locked symbol for which do the acquire lease
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
Screen reader user unable to know that why the locked symbol is provided.

**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference**

[1906_A11y_AzureToolsStorageExplorer_BlobFile_BlobContainerAcquireLease_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3651034/1906_A11y_AzureToolsStorageExplorer_BlobFile_BlobContainerAcquireLease_ScreenReader.pptx)

**Comment by craxal** [comment]
@nirmalsahoo Is the problem that we don't announce the change to item, or that we don't say anything about the lock when the item is selected?

**Comment by craxal** [comment]
Same root cause as #1983.

**Comment by Suraj1200499** [comment]
@Craxal 
1. After selecting Item when hit enter on "Acquire Lease" screen reader not announces anything about the Change to Item. 
2. And also Move to next item and again go back to "Locked" Item, Observe screen reader not announces any information about the Lock.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT  Verified on version 1.12.0 and NVDA 2019.2.1. The issue is still reproduced. NVDA does not read the locked status as after click on acquire lease.NVDA should read the locked status comes after click on acquire lease.

**Comment by craxal** [comment]
So, it looks like the Acquire/Break Lease commands don't trigger the table ARIA label update. I'll need to investigate how to do that.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
